pkgs/development/libraries/mesa/default.nix

```diff
@@ -36,16 +35,16 @@ let
     then ["virgl" "nouveau" "freedreno" "vc4" "etnaviv" "imx"]
     else if stdenv.isAarch64
     then ["virgl" "nouveau" "vc4" ]
-    else ["virgl" "svga" "i915" "r300" "r600" "radeonsi" "nouveau"]);
```

Comment:
Don't really know what to prefer here, read somewhere that the gallium driver is "unofficial" so went with the dri driver.